<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCircleForm</class>
 <widget class="QWidget" name="CreateCircleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>236</width>
    <height>267</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>236</width>
    <height>267</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>158</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnEnterData">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Center X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editCenterX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Center Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editCenterY"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Radius</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editRadius"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
